docs/getting-started.md

```diff
@@ -51,9 +51,11 @@ To create your first DAO run:
 
 ```
 aragon init foo.aragonpm.eth
+cd foo
+aragon run
 ```
 
-Congrats you have just created a DAO! It’s running on your local network and you can check it out by going to [localhost:3001](http://localhost:3001)
+Congrats you have just created a DAO! It’s running on your local network and as soon as it's ready it will open in your browser!
```

Comment:
Should we continue to link to http://localhost:3001 just in case the auto-open browser feature doesn't work for someone?

Reply:
Good idea! Looks like it's port 3000 actually.